Backpack/Card/Classes/BPKDividedCard.m

```diff
@@ -0,0 +1,121 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "BPKDividedCard.h"
+#import "BPKCardDivider.h"
+
+#import <Backpack/Spacing.h>
+#import <Backpack/Color.h>
+#import <Backpack/Radii.h>
+#import <Backpack/Shadow.h>
+
+NS_ASSUME_NONNULL_BEGIN
+@interface BPKDividedCard()
+@property ( strong, nonatomic ) UIStackView *contentView;
+@property ( strong, nonatomic ) BPKCardDivider *lineView;
+@property ( assign, nonatomic ) UIView *primarySubview;
+@property ( assign, nonatomic ) UIView *secondarySubview;
+@end
+
+
+@implementation BPKDividedCard
+
+- (nullable instancetype)initWithCoder:(NSCoder *)aDecoder {
+    self = [super initWithCoder:aDecoder];
+    
+    if (self) {
+        [self setupViews];
+    }
+    
+    return self;
+}
+
+- (instancetype)initWithFrame:(CGRect)frame {
+    self = [super initWithFrame:frame];
+    
+    if (self) {
+        [self setupViews];
+    }
+    
+    return self;
+}
+
+- (instancetype)initWithPadded:(BOOL)padded {
+    self = [super initWithPadded:padded];
+    
+    if (self) {
+        [self setupViews];
+    }
+    
+    return self;
+}
+
+- (instancetype)initWithSubviews:(UIView*)primarySubview withSecondarySubview:(UIView*)secondarySubview {
+    self = [super initWithPadded:true];
+    
+    _primarySubview = primarySubview;
+    _secondarySubview = secondarySubview;
+    
+    if (self) {
+        [self setupViews];
+    }
+    
+    return self;
+}
+
+- (void)setPadded:(BOOL)padded {
+    [super setPadded:padded];
+    self.contentView.spacing = padded ? BPKSpacingBase : 0;
+}
+
+- (void)setOrientation:(UILayoutConstraintAxis)orientation {
+    _orientation = orientation;
+    self.lineView.orientation = orientation;
+    self.contentView.axis = orientation;
+}
+
+- (void)setSubviews:(UIView*)primarySubview withSecondarySubview:(UIView*)secondarySubview {
+    [self.contentView removeArrangedSubview:self.primarySubview];
```

Comment:
This should be guarded by if checks on `self.primarySubView` and `self.secondarySubview`